scripts/parseJson.js

```diff
@@ -0,0 +1,171 @@
+const loadJsonFile = require('load-json-file');
+const writeJsonFile = require('write-json-file');
+
+loadJsonFile('./scripts/tempConfig.json').then(json => {
+	var moduleSidebarList = []
+	var completeModules = []
+
+    json.forEach(function (module) {
```

Comment:
You can use `load.sync` and chain this `forEach` into a simple to understand line.

```js
var moduleSidebarList = []
var completeModules = []
load.sync(path.join(__dirname, 'tempConfig.json')).forEach(module => {
```